Makefile

```diff
@@ -0,0 +1,91 @@
+project := basictracer
+
+pytest := PYTHONDONTWRITEBYTECODE=1 py.test --tb short -rxs \
+	--cov-config .coveragerc --cov $(project) tests
+
+html_report := --cov-report=html
+test_args := --cov-report xml --cov-report term-missing
+
+.PHONY: clean-pyc clean-build docs clean
+.DEFAULT_GOAL : help
+
+help:
```

Comment:
I'm just rubberstamping this Makefile... let me know if you want a real review of it.